Sources/TuistCore/Utils/RootDirectoryLocator.swift

```diff
@@ -54,7 +54,16 @@ public final class RootDirectoryLocator: RootDirectoryLocating {
     }
 
     public func locate(from path: AbsolutePath) async throws -> AbsolutePath? {
-        try await locate(from: path, source: path)
+        if let path = try await locate(from: path, source: path) {
+            return path
+        } else if try await fileSystem.exists(path, isDirectory: true), try await fileSystem.exists(
```

Comment:
Why are we checking if the `path` exists? Shouldn't this be fine?
```suggestion
        } else if try await fileSystem.exists(
```

Reply:
ahh, just read this comment: https://github.com/tuist/tuist/pull/7156#discussion_r1887578688

We can keep it as-is then 👍